src/app/reference-data/reference-data.component.ts

```diff
@@ -12,7 +12,7 @@ export class ReferenceDataComponent {
   
   openModal(){
     this.modalService.open(ReferenceDataModalComponent, {
-      backdrop: 'static'
+      keyboard: false
```

Comment:
keyboard false is needed?
Aren't we doing this in the HTML?

Reply:
It is, because even with the data-bs-keyboard: false in the HTML, both the reference data and the property management modal were still closing with the 'esc' button. So the fix was to include keyboard: false in the components.

Reply:
This means that "data-bs-keyboard="false" data-bs-backdrop="static"" is not needed anymore. Am i correct?

Reply:
Correct. Removed in the last commit.